ImputationPipeline/PCATasks.wdl

```diff
@@ -102,15 +114,23 @@ task ArrayVcfToPlinkDataset {
     File? subset_to_sites
     String basename
     Int mem = 8
+    Int nthreads = 16
   }
 
-  Int disk_space =  3*ceil(size(vcf, "GB")) + 20
+  Int disk_space =  3 * ceil(size(vcf, "GB")) + 20
 
-  command {
-
-    /plink2 --vcf ~{vcf} --extract-intersect ~{pruning_sites} ~{subset_to_sites} --allow-extra-chr --set-all-var-ids @:#:\$1:\$2 \
-    --new-id-max-allele-len 1000 missing --out ~{basename} --make-bed --rm-dup force-first
-  }
+  command <<<
+    /plink2 \
+      --vcf ~{vcf} \
+      --threads ~{nthreads} \
```

Comment:
does it make sense to put this here, it looks like plink will default to using the number of cores available?

Reply:
https://www.cog-genomics.org/plink/2.0/other

Reply:
Yes, that makes sense. I believe we can trust the plink2 defaults. I'll also remove other similar usages of this.